.travis.yml

```diff
@@ -13,6 +13,7 @@ env:
     - DJANGO_VERSION=2.1
     - DJANGO_VERSION=2.2
     - DJANGO_VERSION=3.0
+    - DJANGO_VERSION=3.1
```

Comment:
It looks like 3.1 is only available in python >= 3.6. We need to exclude it for 3.5.

Unrelated, pypy+1.11 is failing because it can't install `cryptography`. Let's remove the specific include for that environment/version combo, though nothing you did here introduced that problem.

Reply:
You'll have to double-check that I did this correctly, I honestly don't know how to run these tests.

Reply:
They're passing now, thank you. You can see it in the checks section at the bottom of the pull.

If things were working correctly, travis should've posted a notice saying that they were broken but that didn't happen for some reason. I've migrated the project over to travis-ci.com and shut off the hook for the old travis-ci.org integration.